pkg/smokescreen/acl_loader_v1.go

```diff
@@ -18,41 +18,41 @@ type EgressAclRule struct {
 }
 
 type EgressAclConfig struct {
-	Services map[string]EgressAclRule
-	Default  *EgressAclRule
-	GlobalDenyList []string
+	Services        map[string]EgressAclRule
+	Default         *EgressAclRule
+	GlobalDenyList  []string
 	GlobalAllowList []string
 }
 
-func (ew *EgressAclConfig) Decide(fromService string, toHost string) (EgressAclDecision, bool, error) {
+func (ew *EgressAclConfig) Decide(fromService string, toHost string) (EgressAclDecision, string, bool, error) {
 	var action EgressAclDecision
 
 	rule := ew.ruleForService(fromService)
 	if rule == nil {
 		action = EgressAclDecisionDeny
-		return action, false, nil
+		return action, "", false, nil
 	}
 
 	defaultRuleUsed := rule == ew.Default
 
 	// if the host matches any of the rule's allowed domains, allow
 	for _, domainGlob := range rule.DomainGlob {
 		if hostMatchesGlob(toHost, domainGlob) {
-			return EgressAclDecisionAllow, defaultRuleUsed, nil
+			return EgressAclDecisionAllow, "if the host matches any of the rule's allowed domains, allow", defaultRuleUsed, nil
```

Comment:
I think the phrasing for these should resemble more in the active past tense, like "Destination matched a rule in the role's list of allowed_domains"

(same for the next few reason strings)

Reply:
sgtm, will do.